rock_paper_scissors/api/models.py

```diff
@@ -19,7 +19,7 @@ class Game(ModelBase):
     player_1_id = Column(Integer, ForeignKey('user.id'))
     player_2_id = Column(Integer, ForeignKey('user.id'))
     rounds_limit = Column(Integer)
-    winner_id = Column(Integer, ForeignKey('user.id'), nullable=False)
+    winner_id = Column(Integer, ForeignKey('user.id'), nullable=True)
```

Comment:
Will I want to create `nullable` parameters in the other Columns of the models for the module that don't have ForeignKeys?